src/index.js

```diff
@@ -1,14 +1,24 @@
+// eslint-disable-next-line no-unused-vars
 import _ from 'lodash';
+import './reset.css';
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const logBoard = [];
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+function render() {
+  const leaderBoard = document.getElementById('JS-score-log');
+  leaderBoard.innerHTML = '';
+  const sortedTask = logBoard.sort((a, b) => a.index - b.index);
```

Comment:
- [ ] Kindly change regular function to arrow function as stated in the project requirements particularly the JavaScript best practices section, [here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.)